.circleci/config.yml

```diff
@@ -71,28 +66,36 @@ commands:
           command: |
             curl -fLSs \
             https://raw.githubusercontent.com/CircleCI-Public/circleci-cli/master/install.sh | sudo bash
+  quality:
+    description: "Run quality"
+    steps:
+      - set_up_environment
+      - run_with_languages:
+          label: Run quality
+          command: |
+            bundle exec rake quality
 
 jobs:
-  overcommit:
+  quality:
+    working_directory: ~/quality
     docker:
       - image: apiology/circleci-ruby:latest
     steps:
       - when:
           condition:
             equal: [<< pipeline.git.branch >>, "main"]
           steps:
-            - overcommit
+            - quality
       - unless:
           condition:
             equal: [<< pipeline.git.branch >>, "main"]
           steps:
-            - run: echo "overcommit only runs on main branch"
+            - run: echo "quality only runs on main branch"
+            - quality  # TODO
   build:
```

Comment:
[punchlist] Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker